deploy/examples/create-external-cluster-resources.py

```diff
@@ -58,13 +61,9 @@
     # for 3.x
     from urllib.parse import urlparse
 
-py3k = False
 try:
-    from urlparse import urlparse
     from base64 import encodestring
 except:
-    py3k = True
-    from urllib.parse import urlparse
     from base64 import encodebytes as encodestring
```

Comment:
actually this is also only for python3 that why I am saying to keep the check for python3 here 

AN=nd remove from above